### ISSUE TITLE
yarn why does not tell if the package is a peer dependency

### ISSUE THREAD

**Issue by villesau** [yarnpkg/yarn#3301]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
`yarn why` does not tell if the reason behind dependency is actually peer-dependency

**If the current behavior is a bug, please provide the steps to reproduce.**
have a dependency with peer-dependency (ie. sass-loader with node-sass)
ask `yarn why node-sass`
the result is following:

```
yarn why node-sass
yarn why v0.23.3
[1/4] 🤔  Why do we have the module "node-sass"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
info Has been hoisted to "node-sass"
info This module exists because it's specified in "dependencies".
info Disk size without dependencies: "7.71MB"
info Disk size with unique dependencies: "12.09MB"
info Disk size with transitive dependencies: "37.87MB"
info Amount of shared dependencies: 109

```

**What is the expected behavior?**
I'd like to know that node-sass exists also because it's a peer dependency of sass-loader

**Please mention your node.js, yarn and operating system version.**
yarn: v0.23.3
node.js: v6.5.0
os: Latest osx

**Comment by bestander** [comment]
Yarn won't install peerDependnecy if it is not present at top level as a dependency.
You have `This module exists because it's specified in "dependencies".` and that is why it is installed.
I guess `why` could also list what packages have it as peer dependency.

**Comment by villesau** [comment]
true. But the actual dependency might exist only because it's a peer dependency of some other package :)
